packages/react-testing/README.md

```diff
@@ -665,6 +665,10 @@ Enzyme is a very popular testing library that heavily inspired the approach this
 
 While the premise of writing tests that mirror user actions is compelling, basing all tests off the raw DOM being produced becomes unmanageable for larger apps.
 
+### What versions of React does this support?
+
+The React versions this library supports are spelled out via a [peer dependency in the package.json](https://github.com/Shopify/quilt/blob/main/packages/react-testing/package.json#L47-L47)
```

Comment:
Looks like we need to update the react/react-dom peer deps to `"^16.8.0 || ^17.0.0"`